jenkins/scripts/bare_metal_lab/deploy-lab.yaml

```diff
@@ -56,19 +57,43 @@
         version: master
       tags: git
 
+    - name: Power off BMHs
```

Comment:
nit: this was a bit confusing for me, because it is not yet a BMH (k8s CRD) but actual server.

Reply:
Right, I understand the confusion. I was thinking that the point is really to make sure that the BMHs from the previous pipeline are cleaned up. At this point the management cluster is still running with these BMHs registered, so in that sense they are BMHs.